app/metrics.yaml

```diff
@@ -809,6 +809,61 @@ sync_auth:
     notification_emails:
       - fenix-core@mozilla.com
     expires: "2020-03-01"
+  sign_up:
+    type: event
+    description: >
+      User registered a new Firefox Account, and was signed into it
+    bugs:
+      - 4971
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/4931#issuecomment-529740300
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-03-01"
+  paired:
+    type: event
+    description: >
+      User signed into FxA by pairing with a different Firefox browser, using a QR code
+    bugs:
+      - 4971
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/4931#issuecomment-529740300
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-03-01"
+  auto_login:
+    type: event
+    description: >
+      User signed into FxA via an account shared from another locally installed Mozilla application (e.g. Fennec)
+    bugs:
+      - 4971
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/4931#issuecomment-529740300
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-03-01"
+  recovered:
+    type: event
+    description: >
+      Account manager automatically recovered FxA authentication state without direct user involvement
+    bugs:
+      - 4971
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/4931#issuecomment-529740300
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-03-01"
+  other_external:
+    type: event
+    description: >
+      User signed into FxA via an unknown mechanism
```

Comment:
All non-unknown methods that are tracked via telemetry are listed above. Unknown in this context is "fxa added a new 'auth action', which is not signin/signup/pairing, but client code doesn't know about it yet".

Reply:
Would prefer if you could document that in this probe description, or explain how to determine the unknown mechanism (and I know there's a tradeoff here, for it going out of date, so maybe the latter) - if someone is reading only the documentation to understand what this probe is, and esp if they are Product or Data Science, it can be confusing.